### DISCUSSION TITLE
shellcheck setup in Dot extra

### DISCUSSION THREAD

**Discussion by matmut7** [LazyVim/LazyVim#3528] [Q&A] [answered]
In the Markdown extra, `markdownlint` is installed using Mason and is optionally set up in [nvim-lint](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/lang/markdown.lua#L37) and [none-ls](https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/plugins/extras/lang/markdown.lua#L28).

In the Dot extra, `shellcheck` is installed using Mason but it is not set up. Why? If this is a mistake, I'd be glad to contribute the changes!

**Comment by folke** [comment, marked as answer]
Because we use bashls for linting etc and that uses shellcheck under the hood

**Reply by matmut7** [reply]
Thanks, didn't see that

**Reply by aungbu17** [reply]
Thanks, didn't see that
